sporkfish/searcher/searcher_config.py

```diff
@@ -0,0 +1,46 @@
+from enum import Enum
+
+from ..configurable import Configurable
+
+
+class SearchMode(Enum):
+    SINGLE_PROCESS = "SINGLE_PROCESS"
+    LAZY_SMP = "LAZY_SMP"
+
+
+class MoveOrderMode(Enum):
```

Comment:
Sorry - could we remove these into move_ordering.py?

Reply:
done